pysim/compositesystem.pyx

```diff
@@ -25,22 +25,54 @@ cdef class CompositeSystem(SimulatableSystem):
         self.connections = Connections._create(&_c_sys_local.connectionHandler)
         self.subsystems = {}
         self.res = Results._create(_c_sys_local.getStoreHandlerP())
+        self.pars = Par()
 
     def __dealloc__(self):
         del self._c_sys
-
+        
+    def __iter__(self):
```

Comment:
I don't think this should be here. Let's use the code below instead as it is more clear what it does.
`for system in sys.subsystems.values():`